include/pops/simulation.hpp

```diff
@@ -190,38 +190,57 @@ class Simulation
         }
     }
 
+    /** kills infected hosts based on mortality rate and timing
+     *
```

Comment:
Please add a paragraph about mortality_rate and mortality_tracker_vector, like ...its size needs to be at least `mortality_time_lag + 1` ... first item is the longest infected ... and all hosts in it die regardless of mortality_rate.